### PR TITLE
Make code loadable

### PR DESCRIPTION
**Pull request by pdbreen**

### CODE REVIEW

File: src/Models/Payment.php [hunk 1]
```diff
@@ -13,8 +13,6 @@ class Payment extends BaseModel
     protected $casts = [
     ];
 
-    protected $refund = app('refund');
```
Comment (pdbreen) [inline src/Models/Payment.php:16]: This is not legal PHP - can't use a function in member assignment.  I'm updating the checkout packages to require their true dependencies and stumbled on this trying to run tests.

File: src/Models/Payment.php [hunk 2]
```diff
@@ -94,7 +92,9 @@ public function generateAmountRefunded()
      */
     public function requestRefund($amount = null, $method = 'Stripe')
     {
-        return $this->refund::create([
+        $refundModel = app('refund');
```
Comment (pdbreen) [inline src/Models/Payment.php:95]: You can add `/** @var Model $refundModel */` above and it should be ok.  Per discussion yesterday, this package should add an explicit dependency on the refund package and then use the Refund model directly and then work to move that dependency into a service / model interface later.

Comment (drewroberts) [reply src/Models/Payment.php:95]: Thanks

File: src/Models/Payment.php [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?php namespace TipOff\Payments\Models;
 
+use Illuminate\Database\Eloquent\Model;
```
Comment (drewroberts) [inline src/Models/Payment.php:3]: Thank you. That makes sense.

**Review by drewroberts — APPROVED** [review]
Thank you, Patrick! 🛃

**Pull request merged**